gulp/tasks/copy.js

```diff
@@ -10,9 +10,12 @@ function copy() {
         systemjs = 'jspm_packages/system.js*';
     }
 
+    polyfilljs = 'jspm_packages/system-polyfills.js';
```

Comment:
`polyfilljs` is undefined, which is causing travis to error. It should probably be added to line 5 as well.